src/pages/index.mdx

```diff
@@ -22,10 +22,12 @@ This section walks you through building on Stellar without smart contracts. Lear
 
 ### Smart contracts
 
-Smart contracts on Stellar launched on Mainnet following a successful validator vote on February 20th, 2024. Stellar's smart contract platform includes the smart contract environment, a Rust SDK, a CLI, and an RPC server. 
+Smart contracts on Stellar launched on Mainnet following a successful validator vote on February 20th, 2024. Stellar's smart contract platform, known as Soroban, includes the smart contract environment, a Rust SDK, a CLI, and an RPC server.
```

Comment:
I have been thinking about the way we use the term Soroban to more narrowly refer to the contract runtime, rather than a platform as a whole. I might be thinking about this incorrectly. But for example, Stellar is a platform because it provides the entire stack, and the Stellar platform uses the Soroban runtime to run contracts. In this world the only things labeled as "soroban" are the runtime itself (which we call the Soroban Environment) and the soroban Rust SDK which is used to build contracts to run on the Soroban Enviroment. Everything else surrounding these things are I think not Soroban specific but part of the Stellar platform. For example, the CLI, the RPC, while they started as Soroban specific tools, they are more broadly part of the Stellar platform and if another platform adopted Soroban they'd really need to dev their own CLI and RPC.

But I say all this not knowing if the nomenclature I'm using is accurate.

cc @tomerweller

Reply:
I think Soroban historically has been referring to the smart contract platform. If we are renaming to only the contract runtime, which I don't oppose, we may need to also think about some of the marketing language on stellar.org, as I think that has wider impact than dev docs.

Reply:
I think the end goal should be what @leighmcculloch describes - Soroban is the name of the runtime (or "environment", or "virtual machine" to align with evm and svm). 

I recognize that in a lot of places the current scope is much wider and I don't think it's the end of the world for now, especially while tools like soroban-cli and soroban-rpc haven't been refactored yet.

Reply:
> to align with evm

We could use this comparison explicitly as a way to help people understand what Soroban is. Soroban == EVM. Rust == Solidity. Stellar == Ethereum.

Reply:
@briwylde08 FYI on `soroban` usage. 

Closing this PR as soroban definition will be in the glossary.